resources/assets/lib/globals.d.ts

```diff
@@ -122,9 +122,9 @@ interface Country {
 }
 
 interface Cover {
-  custom_url?: string;
-  id?: string;
-  url?: string;
+  custom_url: string | null;
+  id: string | null;
+  url: string | null;
```

Comment:
the one from backend always returns a url...

Reply:
`UserNotFound` for the user card uses `null`